### PR TITLE
Increase eslint coverage to client js and tests

### PR DESCRIPTION
**Pull request by HughePaul**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,13 +1,13 @@
 {
   "name": "hmpo-frontend-toolkit",
-  "version": "4.0.0",
+  "version": "4.1.0",
```
Comment (gavboulton) [inline package.json]: This is odd, the module is already at 4.1.0.

**Comment by gavboulton** [comment]
This looks fine to me.

File: package.json (hunk 1 above)
Comment (easternbloc) [inline package.json]: ??? I thought that version bumps were done outside of PR's?

**Comment by easternbloc** [comment]
Other than my version comment LGTM

File: package.json (hunk 1 above)
Comment (HughePaul) [inline package.json]: Master branch says 4.1.0 already, not sure why github has this in the diff.

**Comment by easternbloc** [comment]
I assume a rebase fixed it. LGTM

**Comment by daniel-ac-martin** [comment]
LGTM.

**Pull request merged**